app/core/mapUtils.js

```diff
@@ -1,6 +1,58 @@
 'use strict';
 
 import _ from 'lodash';
+import path from 'path';
+
+// Add proj4 extension to leaflet for historical maps.
+// Don't change the import order!
+import L from 'leaflet';
+import 'proj4leaflet';
+
+import {TILE_LAYERS} from '../constants/MapConstants';
+
+L.Map.prototype.setCrs = function(newCrs) {
+  this.options.crs = newCrs;
+};
+const leafletImageFolder = path.resolve(__dirname, '../../node_modules/leaflet/dist/images/');
+L.Icon.Default.imagePath = leafletImageFolder;
+
+const crs = _makeCRS();
+const layers = _makeLayers();
+
+function _getGeoerverUrl(layerName, layerFmt) {
+  /*eslint-disable */
+  return `http://geoserver.hel.fi/geoserver/gwc/service/tms/1.0.0/${layerName}@ETRS-GK25@${layerFmt}/{z}/{x}/{y}.${layerFmt}`;
+  /*eslint-enable */
+}
+
+function _makeCRS() {
+  const crsName = 'EPSG:3879';
+  /*eslint-disable */
+  const projDef = '+proj=tmerc +lat_0=0 +lon_0=25 +k=1 +x_0=25500000 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs';
+  /*eslint-enable */
+  const bounds = [25440000, 6630000, 25571072, 6761072];
+  const resolutions = [256, 128, 64, 32, 16, 8, 4, 2, 1, 0.5, 0.25, 0.125, 0.0625];
```

Comment:
Maybe it's not needed, because this file is mostly full of that kind of variables.